src/utils/constants.ts

```diff
@@ -20,3 +20,8 @@ import { BILLION } from './numbers'
 
 export const MINIMAL_GAS_AMOUNT = 20000
 export const MINIMAL_GAS_PRICE = BigInt(BILLION * 100) // 100 nanoALPH for the first year to prevent DoS attacks
+
+export const txTypes = {
```

Comment:
I'm not sure that the `constants.ts` file is the best place to define types.
Should we init a `src/types` folder? WDYT @nop33

Reply:
It's not TypeScript types. It's JS string constants.